Test/TestCase/SearchManagerTest.php

```diff
@@ -158,12 +159,49 @@ public function testRemove()
         $this->assertNull($view);
     }
 
+    public function testRemoveAll()
```

Comment:
A test case for multiple entities is easy enough. A timeseries index test would require a new entity, right? Should that be considered part of this task or added as a followup?